src/main/java/com/redhat/devtools/lsp4ij/settings/LanguageServerView.java

```diff
@@ -287,9 +287,7 @@ private JPanel createSettings(JComponent description, boolean launchingServerDef
                 launchingServerDefinition ? LanguageServerPanel.EditionMode.EDIT_USER_DEFINED :
                         LanguageServerPanel.EditionMode.EDIT_EXTENSION, project);
         this.mappingPanel = languageServerPanel.getMappingsPanel();
-        return builder
-                .addComponentFillVertically(new JPanel(), 50)

```

Comment:
I'm not sure why this was added originally, but it was resulting in a significant unusable "padding" in the bottom half (or so) of all tabbed editors. I don't see any negative side-effects of removing it, and it allows the two JSON text editors to expand vertically to fill the available space in the Configuration tab.

Reply:
@SCWells72 this panel is also used in the standard préférences of IJ with Language Servers entre.

If remember I did that to have a correct UI in this preference.

Please check that.

Reply:
Yep, it all looks fine there as well:

![image](https://github.com/user-attachments/assets/bf7c9de5-6b2a-47d2-8ce1-855abb3d00ba)